File: assets/samples/dns.lookup/perl/dns_lookup_apikey.pl
#!/usr/bin/perl

my $domain = 'example.com';
my $apiKey = 'Your DNS Lookup API Key';
my $type = '_all';

my $url = 'https://www.whoisxmlapi.com/whoisserver/DNSService?'
        . "apiKey=$apiKey&domainName=$domain&type=$type"

print get($url);

File: assets/samples/dns.lookup/ruby/dns_lookup_apikey.rb
require 'open-uri'

domain = 'example.com'
apiKey = 'Your DNS Lookup API Key'
type = '_all'

url = 'https://www.whoisxmlapi.com/whoisserver/DNSService?'
    + 'apiKey=' + apiKey + '&domainName=' + domain +  '&type=' + type

puts open(url).read
